apps/services/user-profile/src/app/app.module.ts

```diff
@@ -18,6 +19,7 @@ import { UserProfileModule as UserProfileV2Module } from './v2/user-profile.modu
     }),
     UserProfileModule,
     UserProfileV2Module,
+    ProblemModule,
```

Comment:
Question: Is this the reason that we now return title and type in the BadRequestExceptions ?